--- kdeaccessibility-3.1.91/kmag/kmag.desktop~	2003-08-28 07:29:37.000000000 +0200
+++ kdeaccessibility-3.1.91/kmag/kmag.desktop	2003-09-09 13:58:10.000000000 +0200
@@ -20,3 +20,5 @@
 Name[sv]=Förstoringsglas
 Name[tr]=Büyüteç
 Name[xx]=xxK Magnifierxx
+
+Categories=Qt;KDE;Utility
--- kdeaccessibility-3.1.91/kmousetool/kmousetool/kmousetool.desktop~	2003-08-28 07:29:38.000000000 +0200
+++ kdeaccessibility-3.1.91/kmousetool/kmousetool/kmousetool.desktop	2003-09-09 13:58:56.000000000 +0200
@@ -30,3 +30,5 @@
 Name[sv]=Kmousetool
 Name[tr]=Fare Aracı
 Name[xx]=xxKMouseToolxx
+
+Categories=Qt;KDE;Utility
--- kdeaccessibility-3.1.91/kmouth/kmouth.desktop~	2003-08-01 06:59:40.000000000 +0200
+++ kdeaccessibility-3.1.91/kmouth/kmouth.desktop	2003-09-09 13:59:32.000000000 +0200
@@ -13,3 +13,5 @@
 Name[pt_BR]=KBoca
 Name[sv]=Kmouth
 Name[xx]=xxKMouthxx
+
+Categories=Qt;KDE;Utility
