diff -ruN kdeaccessibility-3.3.92./kttsd/kttsjobmgr/kttsjobmgr.cpp kdeaccessibility-3.3.92/kttsd/kttsjobmgr/kttsjobmgr.cpp
--- kdeaccessibility-3.3.92./kttsd/kttsjobmgr/kttsjobmgr.cpp	2005-02-03 23:23:39.000000000 +0100
+++ kdeaccessibility-3.3.92/kttsd/kttsjobmgr/kttsjobmgr.cpp	2005-02-19 00:39:18.179715640 +0100
@@ -50,7 +50,7 @@
     * the form 'void *init_lib<library name>()  It always returns a
     * new factory object
     */
-    void *init_libkttsjobmgrpart()
+    KDE_EXPORT void *init_libkttsjobmgrpart()
     {
         return new KttsJobMgrFactory;
     }
