diff -urN -x CVS kdeaccessibility.orig/kmag/kmag.desktop kdeaccessibility/kmag/kmag.desktop
--- kdeaccessibility.orig/kmag/kmag.desktop	Tue Mar 23 13:58:47 2004
+++ kdeaccessibility/kmag/kmag.desktop	Sun Apr 25 00:50:39 2004
@@ -20,9 +20,9 @@
 Name[hu]=KMag
 Name[is]=K stækkunargler
 Name[it]=KMagnifier
+Name[nb]=K Forstørrer
 Name[pl]=KMagnifier (Lupa)
 Name[pt]=Lupa
-Name[pt_BR]=K Lupa
 Name[ro]=Lupă
 Name[ru]=Экранная лупа
 Name[sk]=K Lupa
@@ -35,6 +35,7 @@
 Name[zh_CN]=K 放大镜
 Name[zh_TW]=K-放大鏡
 GenericName=Screen Magnifier
+GenericName[ar]=مكبَر الشاشة
 GenericName[ca]=Lupa de la pantalla
 GenericName[cs]=Lupa obrazovky
 GenericName[cy]=KChwyddwr
@@ -52,11 +53,12 @@
 GenericName[is]=Skjástækkunargler
 GenericName[it]=Lente di ingrandimento per lo schermo
 GenericName[ja]=画面拡大鏡
+GenericName[nb]=Skjermforstørrer
 GenericName[nl]=Schermvergroter
 GenericName[nn]=Skjermforstørrar
 GenericName[pl]=Powiększanie fragmentów ekranu
 GenericName[pt]=Lupa
-GenericName[pt_BR]=Amplificador de Tella
+GenericName[pt_BR]=Amplificador de Tela
 GenericName[ro]=Lupă de ecran
 GenericName[ru]=Экранная лупа
 GenericName[sk]=Lupa obrazovky
diff -urN -x CVS kdeaccessibility.orig/kmousetool/kmousetool/kmousetool.cpp kdeaccessibility/kmousetool/kmousetool/kmousetool.cpp
--- kdeaccessibility.orig/kmousetool/kmousetool/kmousetool.cpp	Wed Nov 19 23:20:45 2003
+++ kdeaccessibility/kmousetool/kmousetool/kmousetool.cpp	Wed Apr  7 13:58:44 2004
@@ -553,7 +553,7 @@
 	if (newSettings())
 	{
 		int answer = KMessageBox::questionYesNoCancel (this,
-			i18n("There are unsaved changes in the active module.<br>Do you want to apply the changes before closing the configuration window or discard the changes?"),
+			"<qt>"+i18n("There are unsaved changes in the active module.<br>Do you want to apply the changes before closing the configuration window or discard the changes?"),
 			i18n("Closing Configuration Window"),
 			KStdGuiItem::apply(), KStdGuiItem::discard(), "AutomaticSave");
 		if (answer == KMessageBox::Yes)
@@ -572,7 +572,7 @@
 	if (newSettings())
 	{
 		int answer = KMessageBox::questionYesNoCancel (this,
-			i18n("There are unsaved changes in the active module.<br>Do you want to apply the changes before quitting KMousetool or discard the changes?"),
+			"<qt>"+i18n("There are unsaved changes in the active module.<br>Do you want to apply the changes before quitting KMousetool or discard the changes?"),
 			i18n("Quitting KMousetool"),
 			KStdGuiItem::apply(), KStdGuiItem::discard(), "AutomaticSave");
 		if (answer == KMessageBox::Yes)
diff -urN -x CVS kdeaccessibility.orig/kmousetool/kmousetool/kmousetool.desktop kdeaccessibility/kmousetool/kmousetool/kmousetool.desktop
--- kdeaccessibility.orig/kmousetool/kmousetool/kmousetool.desktop	Tue Mar 23 13:58:49 2004
+++ kdeaccessibility/kmousetool/kmousetool/kmousetool.desktop	Sun Apr 25 00:50:41 2004
@@ -6,6 +6,7 @@
 MiniIcon=kmousetool.png
 DocPath=kmousetool/index.html
 Comment=Clicks the mouse for you, reducing the effects of RSI
+Comment[ar]=ينقر على الفأرة بدلا منك، مُخَفِّفاً عنك آلام RSI
 Comment[ca]=Fa els clics del ratolí­, reduint els efectes del RSI (índex de força relativa)
 Comment[cs]=Kliká za vás myší, omezuje tak efekt RSI
 Comment[cy]=Clicio'r llygoden i chi, wrth esmwytho effeithiau RSI
@@ -24,6 +25,7 @@
 Comment[is]=Smellir músarhnöppunum fyrir þig og dregur úr áhrifum RSI
 Comment[it]=Fa clic sul mouse al posto tuo
 Comment[ja]=RSIの効果を縮小するにはマウスをクリック
+Comment[nb]=Trykker museknappen for deg, gir mindre musesyke
 Comment[nl]=Klikt de muisknop voor u, vermindert de effecten van RSI
 Comment[nn]=Klikkar med musa for deg, for å hindra belastningsskadar
 Comment[pl]=Klika za Ciebie myszą, ograniczając nadwyrężanie mięśni
@@ -47,7 +49,6 @@
 Name[cy]=KErfynLlygoden
 Name[eo]=Kmusilo
 Name[hi]=के-माउस टूल
-Name[pt_BR]=Ferramenta do Mouse
 Name[ro]=Utilitar mouse
 Name[sv]=Kmousetool
 Name[ta]=கே மவுஸ்டூல்
@@ -58,6 +59,7 @@
 Name[zh_CN]=K 鼠标工具
 Name[zh_TW]=K-滑鼠工具
 GenericName=Automatic Mouse Click
+GenericName[ar]=نقر الفأرة التلقائي
 GenericName[ca]=Clic automàtic del ratolí
 GenericName[cs]=Automatické klikání myší
 GenericName[cy]=Clic Llygoden Ymysgogol
@@ -76,6 +78,7 @@
 GenericName[is]=Sjálfvirkur músahnappasmellir
 GenericName[it]=Clic automatico del mouse
 GenericName[ja]=自動マウスクリック
+GenericName[nb]=Automatisk museklikk
 GenericName[nl]=Automatische muisklik
 GenericName[nn]=Automatisk museklikk
 GenericName[pl]=Automatyczne klikanie myszą
diff -urN -x CVS kdeaccessibility.orig/kmouth/books/de.desktop kdeaccessibility/kmouth/books/de.desktop
--- kdeaccessibility.orig/kmouth/books/de.desktop	Sun Feb 29 15:40:26 2004
+++ kdeaccessibility/kmouth/books/de.desktop	Sun Apr 25 00:50:43 2004
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=German
 Name[af]=Duits
-Name[ar]=الألمانية
+Name[ar]=ألمانية
 Name[az]=Almanca
 Name[be]=Нямецкая
 Name[bg]=Немски
@@ -27,6 +27,7 @@
 Name[he]=גרמנית
 Name[hi]= जर्मन
 Name[hr]=Njemački
+Name[hsb]=Němsce
 Name[hu]=Német
 Name[id]=Jerman
 Name[is]=Þýska
diff -urN -x CVS kdeaccessibility.orig/kmouth/books/en.desktop kdeaccessibility/kmouth/books/en.desktop
--- kdeaccessibility.orig/kmouth/books/en.desktop	Tue Mar 23 13:58:51 2004
+++ kdeaccessibility/kmouth/books/en.desktop	Sun Apr 25 00:50:45 2004
@@ -1,6 +1,7 @@
 # KDE Config File  
 [Desktop Entry]
 Name=English
+Name[ar]=إنجليزية
 Name[ca]=Anglès
 Name[cs]=Anglicky
 Name[cy]=Saesneg
@@ -22,6 +23,7 @@
 Name[mn]=Англи
 Name[ms]=Inggeris
 Name[mt]=Ingliż
+Name[nb]=Engelsk
 Name[nl]=Engels
 Name[nn]=Engelsk
 Name[pl]=Angielska
diff -urN -x CVS kdeaccessibility.orig/kmouth/books/sv.desktop kdeaccessibility/kmouth/books/sv.desktop
--- kdeaccessibility.orig/kmouth/books/sv.desktop	Sun Feb 29 15:40:30 2004
+++ kdeaccessibility/kmouth/books/sv.desktop	Sun Apr 25 00:50:47 2004
@@ -2,7 +2,7 @@
 [Desktop Entry]
 Name=Swedish
 Name[af]=Sweedse
-Name[ar]=السويدية
+Name[ar]=سويدية
 Name[az]=İsveçcə
 Name[be]=Швэдзкая
 Name[bg]=Шведски
diff -urN -x CVS kdeaccessibility.orig/kmouth/kmouth.desktop kdeaccessibility/kmouth/kmouth.desktop
--- kdeaccessibility.orig/kmouth/kmouth.desktop	Tue Mar 23 13:58:53 2004
+++ kdeaccessibility/kmouth/kmouth.desktop	Sun Apr 25 00:50:49 2004
@@ -11,12 +11,12 @@
 Comment[xx]=xxxx
 Terminal=0
 Name=KMouth
+Name[ar]=برنامج KMouth
 Name[cy]=KCeg
 Name[eo]=KBuŝo
 Name[hi]=के-माउथ
 Name[is]=KMunnur
 Name[mn]=KМаф(КMouth)
-Name[pt_BR]=KBoca
 Name[ro]=Vorbitor
 Name[sv]=Kmouth
 Name[ta]=K மவுத்
@@ -24,6 +24,7 @@
 Name[xx]=xxKMouthxx
 Name[zh_CN]=K 语音精灵
 GenericName=Speech Synthesizer Frontend
+GenericName[ar]=واجهة النَظام الصَوتي
 GenericName[ca]=Interfície per al sintetitzador de veu
 GenericName[cs]=Rozhraní hlasového synthetizátoru
 GenericName[cy]=Blaen-wyneb Syntheseisydd Lleferydd
@@ -41,6 +42,7 @@
 GenericName[is]=Viðmót á talgerfilinn
 GenericName[it]=Interfaccia grafica per il sintetizzatore vocale
 GenericName[ja]=スピーチシンセサイザフロントエンド
+GenericName[nb]=Brukerflate for talesystem
 GenericName[nl]=Spraaksynthesizer
 GenericName[nn]=Grensesnitt for talesyntese
 GenericName[pl]=Program obsługujący syntezator mowy
diff -urN -x CVS kdeaccessibility.orig/kmouth/phrasebook/phrasebook.cpp kdeaccessibility/kmouth/phrasebook/phrasebook.cpp
--- kdeaccessibility.orig/kmouth/phrasebook/phrasebook.cpp	Mon Oct 27 10:24:35 2003
+++ kdeaccessibility/kmouth/phrasebook/phrasebook.cpp	Sat Apr 24 00:32:23 2004
@@ -302,7 +302,7 @@
          result = save (url, false);
       }
       else {
-         int filetype = KMessageBox::questionYesNoCancel (0,QString("<qt>%1</qt>").arg(i18n("Your chosen filename <i>%1</i> has the extension <i>.phrasebook</i>."
+         int filetype = KMessageBox::questionYesNoCancel (0,QString("<qt>%1</qt>").arg(i18n("Your chosen filename <i>%1</i> has the extension <i>.phrasebook</i>. "
                                                            "Do you wish to save in phrasebook format?").arg(url.filename())),i18n("File Extension"),i18n("As Phrasebook"),i18n("As Plain Text"));
          if (filetype == KMessageBox::Cancel) {
             return 0;
