--- kdeaccessibility-031014/kmag/kmag.desktop.orig	2003-10-16 13:23:54.000000000 +0200
+++ kdeaccessibility-031014/kmag/kmag.desktop	2003-10-16 13:24:20.000000000 +0200
@@ -22,3 +22,4 @@
 Name[tr]=Büyüteç
 Name[xx]=xxK Magnifierxx
 Name[zh_CN]=KDE放大镜
+Categories=Qt;KDE;Utility
--- kdeaccessibility-031014/kmousetool/kmousetool/kmousetool.desktop.orig	2003-10-09 02:05:18.000000000 +0200
+++ kdeaccessibility-031014/kmousetool/kmousetool/kmousetool.desktop	2003-10-16 13:24:35.000000000 +0200
@@ -37,3 +37,4 @@
 Name[tr]=Fare Aracı
 Name[xx]=xxKMouseToolxx
 Name[zh_CN]=KDE鼠标工具
+Categories=Qt;KDE;Utility
--- kdeaccessibility-3.1.91/kmouth/kmouth.desktop~	2003-08-01 06:59:40.000000000 +0200
+++ kdeaccessibility-3.1.91/kmouth/kmouth.desktop	2003-09-09 13:59:32.000000000 +0200
@@ -13,3 +13,5 @@
 Name[pt_BR]=KBoca
 Name[sv]=Kmouth
 Name[xx]=xxKMouthxx
+
+Categories=Qt;KDE;Utility
